File: 1538-maximum-points-you-can-obtain-from-cards/maximum-points-you-can-obtain-from-cards.py
class Solution(object):
    def maxScore(self, cardPoints, k):
        """
        :type cardPoints: List[int]
        :type k: int
        :rtype: int
        """
        s=sum(cardPoints[:k])
        m=s
        n=len(cardPoints)
        for i in range(1,k+1):
            s=s-cardPoints[k-i]+cardPoints[n-i]
            m=s if s>m else m
        return m    

File: 1538-maximum-points-you-can-obtain-from-cards/maximum-points-you-can-obtain-from-cards.java
class Solution {
    public int maxScore(int[] cardPoints, int k) {
        int n=cardPoints.length;
        int s=0;
        for(int i=0;i<k;i++){
            s+=cardPoints[i];
        }
        int m=s;
        for(int i=1;i<k+1;i++){
            s+=cardPoints[n-i]-cardPoints[k-i];
            if(s>m)m=s;
        }
        return m;
    }
}